catboost/libs/eval_result/eval_helpers.cpp

```diff
@@ -217,6 +217,9 @@ void PrepareEval(const EPredictionType predictionType,
                 }
             }
             break;
+        case EPredictionType::Exponent:
+            *result = {CalcExponent(approx[0])};
+            break;
```

Comment:
Let's add EPredictionType::Exponent in this place too:
https://github.com/catboost/catboost/blob/master/catboost/libs/model/eval_processing.h#L158-L165